pom.xml

```diff
@@ -41,7 +41,8 @@
     <spring.version>4.1.5.RELEASE</spring.version>
     <fcrepo.version>4.4.0</fcrepo.version>
     <fcrepo-audit.version>4.4.0</fcrepo-audit.version>
-    <fcrepo-camel.version>4.4.0</fcrepo-camel.version>
+    <fcrepo-camel.version>4.4.1-SNAPSHOT</fcrepo-camel.version>
```

Comment:
I would highly recommend keeping this in line with non-SNAPSHOT releases -- it means we can version the projects independently, which is highly desirable. If you would like to have a new `fcrepo-camel` release (with the new `fcrepo-java-client`), I can do that early next week.